dbld/images/centos7/Dockerfile

```diff
@@ -3,11 +3,13 @@ MAINTAINER Andras Mitzki <andras.mitzki@balabit.com>
 
 RUN yum install -y wget epel-release
 
-RUN cd /etc/yum.repos.d && wget https://copr.fedorainfracloud.org/coprs/czanik/syslog-ng39/repo/epel-7/czanik-syslog-ng39-epel-7.repo
+RUN cd /etc/yum.repos.d && wget https://copr.fedorainfracloud.org/coprs/czanik/syslog-ng311/repo/epel-7/czanik-syslog-ng311-epel-7.repo
 
 ADD dev-dependencies.txt .
 RUN cat dev-dependencies.txt | grep -v "#" | xargs yum -y  install
-
+ADD gradle_installer.sh .
+RUN ./gradle_installer.sh
+RUN find / -name libjvm.so | xargs dirname | tee --append /etc/ld.so.conf.d/openjdk-libjvm.conf && ldconfig
```

Comment:
What is the reason we run find here? On centos, isn't this file on a fix location? Maybe we can hardcode the path here.